docs/getting-started.md

```diff
@@ -76,13 +76,13 @@ with a Ubuntu 22.04 rootfs from our CI:
 ARCH="$(uname -m)"
 
 # Download a linux kernel binary
-wget https://s3.amazonaws.com/spec.ccfc.min/firecracker-ci/v1.6/${ARCH}/vmlinux-5.10.198
+wget https://s3.amazonaws.com/spec.ccfc.min/firecracker-ci/v1.7/${ARCH}/vmlinux-5.10.198
```

Comment:
before we merge the PR we should get this path active, right now this returns an error

Reply:
Good catch. I will do as a follow-up since it's a doc change that should be fast to pass PR.